### PR TITLE
Normalize CPU stats

### LINKED ISSUE #2131: Performance metrics in the Supervisor UI
## The problem

Performance metrics in the Supervisor UI is reporting more than 100% CPU usage 

## Environment

 Supervisor
Version
247

arch | aarch64
-- | --
chassis | embedded
dev | false
docker | true
docker_version | 19.03.11
hassio | true
host_os | HassOS 4.13
installation_type | Home Assistant OS
os_name | Linux
os_version | 4.19.127-v8
python_version | 3.8.5
supervisor | 247
timezone | America/Detroit
version | 0.116.3
virtualenv | false

Screenshot 2020-10-15 171734

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

Normalize docker CPU stats

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (which adds functionality to the supervisor)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #2131
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] Local tests pass. **Your PR cannot be merged unless tests pass**
- [x] There is no commented out code in this PR.
- [x] I have followed the [development checklist][dev-checklist]
- [x] The code has been formatted using Black (`black --fast supervisor tests`)
- [x] Tests have been added to verify that the new code works.

If API endpoints of add-on configuration are added/changed:

- [ ] Documentation added/updated for [developers.home-assistant.io][docs-repository]

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[docs-repository]: https://github.com/home-assistant/developers.home-assistant

### CODE REVIEW

File: supervisor/docker/stats.py [hunk 1]
```diff
@@ -45,12 +45,9 @@ def _calc_cpu_percent(self, stats):
             stats["cpu_stats"]["system_cpu_usage"]
             - stats["precpu_stats"]["system_cpu_usage"]
         )
-        online_cpu = stats["cpu_stats"]["online_cpus"]
 
-        if online_cpu == 0.0:
-            online_cpu = len(stats["cpu_stats"]["cpu_usage"]["percpu_usage"])
         if system_delta > 0.0 and cpu_delta > 0.0:
-            self._cpu = (cpu_delta / system_delta) * online_cpu * 100.0
+            self._cpu = (cpu_delta / system_delta) * 100.0
```
Comment (pvizeli) [inline supervisor/docker/stats.py:50]: should we add an else self._cpu = 0.0?

Comment (ludeeus) [reply supervisor/docker/stats.py:50]: in what case would that be needed?

Comment (ludeeus) [reply supervisor/docker/stats.py]: I guess if everything is 0 :shrug: 
Added guard and updated test

File: supervisor/docker/stats.py [hunk 2]
```diff
@@ -45,12 +45,12 @@ def _calc_cpu_percent(self, stats):
...
 
-        if online_cpu == 0.0:
-            online_cpu = len(stats["cpu_stats"]["cpu_usage"]["percpu_usage"])
-        if system_delta > 0.0 and cpu_delta > 0.0:
-            self._cpu = (cpu_delta / system_delta) * online_cpu * 100.0
+        self._cpu = (
+            (cpu_delta / system_delta) * 100.0
+            if system_delta > 0.0 and cpu_delta > 0.0
+            else 0.0
+        )
```
Comment (pvizeli) [inline supervisor/docker/stats.py:53]:
```suggestion
       if system_delta > 0.0 and cpu_delta > 0.0:
            self._cpu = (cpu_delta / system_delta) * 100.0
       else:
             self._cpu = 0.0
```
Is a bit better to read

**Pull request merged**